src/note/models.py

```diff
@@ -59,6 +59,7 @@ class Meta:
         User,
         verbose_name=_('Author'),
         null=True,
+        on_delete=models.CASCADE
```

Comment:
Perhaps models.SET_NULL? Although this may not happen in reality - unless an employee is fired...